pkg/operator/ceph/csi/spec.go

```diff
@@ -75,12 +80,15 @@ const (
 	DefaultSnapshotterImage = "quay.io/k8scsi/csi-snapshotter:v1.2.0"
 
 	// template
-	DefaultRBDPluginTemplatePath            = "/etc/ceph-csi/rbd/csi-rbdplugin.yaml"
-	DefaultRBDProvisionerSTSTemplatePath    = "/etc/ceph-csi/rbd/csi-rbdplugin-provisioner-sts.yaml"
-	DefaultRBDProvisionerDepTemplatePath    = "/etc/ceph-csi/rbd/csi-rbdplugin-provisioner-dep.yaml"
+	DefaultRBDPluginTemplatePath         = "/etc/ceph-csi/rbd/csi-rbdplugin.yaml"
+	DefaultRBDProvisionerSTSTemplatePath = "/etc/ceph-csi/rbd/csi-rbdplugin-provisioner-sts.yaml"
```

Comment:
@Madhu-1 do we need this STS path any more ?

Reply:
this is not added in this PR, this is just a rearrangement, STS Path is needed to deploy statefulset if kube version is 1.13

Reply:
ah.. we still need to support `1.13`.